### ISSUE TITLE
Works only with |noescape latte filter

### ISSUE THREAD

**Issue by reflexator** [contributte/latte-parsedown-extra#2]
Hi,

when I call parsedown like this
```
{var $test = '**test**'}
{$test|parsedown}
```
Result is correctly
```
<p><strong>test</strong></p>
```

But with escaped HTML tags. I have to use filter |noescape to print HTML tags correctly

**Comment by f3l1x** [comment]
Hmm, I think I can fix it. Could you please test `contributte/latte-parsedown-extra: dev-master`?

**Comment by f3l1x** [comment]
Should be fixed in v1.3.0
